core/node/node_framework/src/wiring_layer.rs

```diff
@@ -30,9 +30,9 @@ impl fmt::Debug for dyn WiringLayer {
 #[non_exhaustive]
 pub enum WiringError {
     #[error("Layer attempted to add resource {0}, but it is already provided")]
-    ResourceAlreadyProvided(ResourceId),
+    ResourceAlreadyProvided(String),
     #[error("Resource {0} is not provided")]
-    ResourceLacking(ResourceId),
+    ResourceLacking(String),
```

Comment:
If we treat errors are something that node framework users would actually use (i.e., having utility besides converting an error to a string), having `String` variants (i.e., only covering human-readability) is suboptimal. May make sense to add `ResourceId` as well.